README.md

```diff
@@ -537,6 +537,30 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='destructure-inline-let'></a> (<a href='#destructure-inline-let'>link</a>) **When destructuring an enum case or a tuple, place the `let` keyword inline, adjacent to each individual property assignment.** [![SwiftFormat: hoistPatternLet](https://img.shields.io/badge/SwiftFormat-hoistPatternLet-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#hoistPatternLet)
+
+  <details>
```

Comment:
Let's add the reasoning to the rule itself too 🙏

Reply:
Do you think we should include the contrived third code example in the style guide document as well? I'm on the fence about that, because it feels a little too contrived to be worth including (but I wanted to include it in the PR for completeness, since it was brought up in the discussion on Slack)